spec/features/admin/proposals_spec.rb

```diff
@@ -7,6 +7,14 @@
     login_as(admin.user)
   end
 
+  scenario 'Disabled with a feature flag' do
+    Setting['feature.proposals'] = nil
+    admin = create(:administrator)
+    login_as(admin.user)
+
+    expect{ visit admin_proposals_path }.to raise_exception(FeatureFlags::FeatureDisabled)
```

Comment:
Since rspec randomizes the order of the specs, and it doesn't reset the Settings values after each scenario, resetting any Settings value changed at the end of the scenario is the best idea. Check for example https://github.com/consul/consul/blob/master/spec/features/valuation/budget_investments_spec.rb#L11-L16